src/components/tasks/card/card.module.scss

```diff
@@ -5,6 +5,22 @@ $yellow: #ecef08;
 $white: #e5e5e5;
 $black: #252525;
```

Comment:
And this black :O

Reply:
eerie black ,should I change the names accordingly?

Reply:
scss color variables can be done with new PR